### ISSUE TITLE
Getting a 400 error on a release action

### ISSUE THREAD

**Issue by abstractalgo** [opengisch/qgis-plugin-ci#227]
Hello, we're using `qgis-plugin-ci` in our Github Actions, and it worked flawlessly so far. Today, we experienced this:

```sh
2023-06-01 10:06:42||ERROR||release||=== A protocol error occurred ===
Plugin archive created: hidden_name.zip (2.05 Mo)

URL: ***:******@plugins.qgis.org:443/plugins/RPC2/

HTTP/HTTPS headers: {'Date': 'Thu, 01 Jun 2023 10:06:42 GMT', 'Content-Type': 'text/html; charset=utf-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Vary': 'Cookie', 'CF-Cache-Status': 'DYNAMIC'}

Error code: 400

Error message: Bad Request
```

Nothing significant changed since we last released a plugin this way.

```yaml
- name: Deploy plugin
   run: qgis-plugin-ci release ${GITHUB_REF/refs\/tags\//} --github-token ${{ secrets.GITHUB_TOKEN }} --disable-submodule-update --osgeo-username ${{ secrets.OSGEO_USERNAME }} --osgeo-password ${{ secrets.OSGEO_PASSWORD }}
```

I am not sure if QGIS [RPC API](https://plugins.qgis.org/plugins/RPC2/) changed in the meantime perhaps ,or maybe the fact that the plugin had ~300KB before but now sits at ~2MB, is causing issues.

**Comment by Gustry** [comment]
It's indeed a known issue for me as well for a long time.
I tried a PR about this issue : #221 but I forgot to get back to it.

**Comment by abstractalgo** [comment]
Thanks for pointing to the PR! @Gustry 

It seems it still doesn't help us understand what actually caused the issue, right? Or what we should change on our side to mitigate the problem?

**Comment by Gustry** [comment]
Indeed, it doesn't fix the problem.
IMHO, a ticket should be open on the "qgis-django" repository : https://github.com/qgis/QGIS-Django

* Maybe look to logs on the server when the RPC api is used ?
* Try with a "fake" plugin with different zip size ?

**Comment by 3nids** [comment]
I believe this is fixed now thanks to #259
